react/features/base/media/reducer.js

```diff
@@ -87,6 +90,11 @@ const VIDEO_INITIAL_MEDIA_STATE = {
  */
 function _video(state = VIDEO_INITIAL_MEDIA_STATE, action) {
     switch (action.type) {
+    case CLEAR_VIDEO_TRANSFORMS:
+        return {
+            ...state,
+            transforms: {}
+        };
```

Comment:
I guess there's another `case` which needs to be separated with a new line from the totally unrelated `case` bellow it.

Reply:
I'm not sure what you mean here. Can you please explain?

Reply:
It's another comment that I fixed myself eventually.